openpype/hosts/max/plugins/publish/extract_max_scene_raw.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pyblish.api
+from openpype.pipeline import publish
+from pymxs import runtime as rt
+from openpype.hosts.max.api import (
+    maintained_selection,
+    get_all_children
+)
+
+
+class ExtractMaxSceneRaw(publish.Extractor):
+    """
+    Extract Raw Max Scene with SaveSelected
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.2
+    label = "Max Scene(Raw)"
+    hosts = ["max"]
+    families = ["camera"]
+
+    def process(self, instance):
+        container = instance.data["instance_node"]
+
+        # publish the raw scene for camera
+        self.log.info("Extracting Camera ...")
+
+        stagingdir = self.staging_dir(instance)
+        filename = "{name}.max".format(**instance.data)
+
+        max_path = os.path.join(stagingdir, filename)
+        self.log.info("Writing max file '%s' to '%s'" % (filename,
+                                                         max_path))
+
+        if "representations" not in instance.data:
+            instance.data["representations"] = []
+
+        # add extra blacklash for saveNodes in MaxScript
+        re_max_path = stagingdir + "\\\\" + filename
```

Comment:
if this is done for double backslashes, maybe something like:
```python
re_max_path = os.path.normpath(max_path).replace("\\", "\\\\")
```
to be sure that they are everywhere?

Reply:
Most apps also deal with it fine if you force forward slashes instead - it usually makes the output easier to read so if it works I'd recommend: 
```python
os.path.normpath(max_path).replace("\\", "/")
```